app/_hub/kong-inc/proxy-cache/index.md

```diff
@@ -42,61 +42,71 @@ params:
     - name: response_code
       required:
       default: 200, 301, 404
-      value_in_examples:
+      value_in_examples: [ "200" ]
+      datatype: array of type integer
```

Comment:
this way it's consistent with the already existing convention of `array of string elements`

```suggestion
      datatype: array integer elements
```